### PR TITLE
[RDY] vim-patch: mark 7.4.615 as included

### PR DESCRIPTION
**Pull request by ghost**
See https://github.com/neovim/neovim/pull/1761#issuecomment-72737699

Original vim patch: http://ftp.vim.org/vim/patches/7.4/7.4.615

### CODE REVIEW

File: src/nvim/version.c [hunk 1]
```diff
@@ -169,7 +169,7 @@ static int included_patches[] = {
   //618,
   //617,
   //616,
-  //615,
+  //615 NA
```
Comment (oni-link) [inline src/nvim/version.c]: Why not just mark this patch as included?

Comment (justinmk) [inline src/nvim/version.c]: :+1:

Comment (Pyrohh) [inline src/nvim/version.c]: @oni-link I missed your comment at the top of #1761 showing the original patch, so I assumed that it was completely authored by you, so I'll fix.

**Comment by Pyrohh** [comment]
repushed

**Comment by oni-link** [comment]
The vim-patch also updates comments for `list_free()` and `dict_free()`, but original and updated comments are half of the time wrong (with respect to the `recurse` argument). At least that is how I understand these comments.

Otherwise, LGTM.

**Comment by Pyrohh** [comment]
Thanks @oni-link, marked as RDY.

**Comment by Pyrohh** [comment]
ping @justinmk

**Comment by justinmk** [comment]
merged

**Pull request closed without merging**